ObservatoryCore/Utils/AudioHandler.cs

```diff
@@ -9,6 +10,30 @@ internal class AudioHandler : ConcurrentQueue<KeyValuePair<Guid, string>>
         
         private List<Guid> audioTasks = [];
 
+        public void InstantlyPlay(string filePath)
+        {
+            if (new FileInfo(filePath).Length > 0)
+                try
+                {
+                    using (var file = new AudioFileReader(filePath))
+                    using (var output = new WaveOutEvent() { DeviceNumber = Properties.Core.Default.AudioDevice })
```

Comment:
What happens if a user unplugs a USB audio device while playing and device indices change?